ludwig/data/dataframe/dask.py

```diff
@@ -129,7 +149,24 @@ def to_ray_dataset(self, df):
         return from_dask(df)
 
     def from_ray_dataset(self, dataset) -> dd.DataFrame:
-        return dataset.to_dask()
+        """Custom Ray to Dask conversion implementation to pass in meta during dd.DataFrame creation."""
```

Comment:
@geoffreyangus should we revert back to `dataset.to_dask()` if the empty partitions issue doesn't stem from a need to pass in meta?

Reply:
Yup, reverted!